plugins/jobs/src/js/components/form/helpers/MetronomeJobValidators.ts

```diff
@@ -513,19 +393,6 @@ export const MetronomeSpecValidators: MetronomeValidators = {
     return errors;
   },
 
-  argsAreArray(formData: JobOutput) {
```

Comment:
Due to a safeguard in the `argsReducer` this can not be set and not an Array, so i was not able to ever see that message in the UI - thus this method does not seem to add any value and has been removed.

Reply:
that's the only way i tried to break it (and provoke the validation-message). it neither broke, nor did the message show up. just double checked this by reintroducing this validator.

Reply:
Ok cool - looks good as is! Thanks for checking!!